website/pages/downloads/index.jsx

```diff
@@ -1,14 +1,15 @@
 import VERSION, { packageManagers } from 'data/version.js'
 import HashiHead from '@hashicorp/react-head'
 import { productName, productSlug } from 'data/metadata'
-import ProductDownloader from '@hashicorp/react-product-downloader'
+import ProductDownloadsPage from '@hashicorp/react-product-downloads-page'
+import { generateStaticProps } from '@hashicorp/react-product-downloads-page/server'
 import styles from './style.module.css'
 
 export default function DownloadsPage({ releases }) {
   return (
     <span className={styles.downloads}>
       <HashiHead title={`Downloads | ${productName} by HashiCorp`} />
-      <ProductDownloader
+      <ProductDownloadsPage
```

Comment:
wrapping span and head can be dropped here since this is now a full page component

Reply:
Aha I knew I migrated another usage somewhere...thank you 🙏